diff --git a/node_modules/@searchkit/elastic-ui/lib/esm/SearchBar/index.js b/node_modules/@searchkit/elastic-ui/lib/esm/SearchBar/index.js
index 84fb460..ca04460 100644
--- a/node_modules/@searchkit/elastic-ui/lib/esm/SearchBar/index.js
+++ b/node_modules/@searchkit/elastic-ui/lib/esm/SearchBar/index.js
@@ -4,11 +4,11 @@ import React from 'react';
 export const SearchBar = ({ loading }) => {
     const [query, setQuery] = useSearchkitQueryValue();
     const api = useSearchkit();
-    return (React.createElement(EuiFieldSearch, { placeholder: "Search", value: query, onChange: (e) => {
+    return (React.createElement(EuiFieldSearch, { placeholder: "Etsi", value: query, onChange: (e) => {
             setQuery(e.target.value);
         }, isLoading: loading, onSearch: (value) => {
             setQuery(value);
             api.setQuery(value);
             api.search();
-        }, isClearable: true, "aria-label": "Search" }));
+        }, isClearable: true, "aria-label": "Etsi" }));
 };
