diff --git a/node_modules/@searchkit/sdk/lib/esm/transporters/FetchClientTransporter.js b/node_modules/@searchkit/sdk/lib/esm/transporters/FetchClientTransporter.js
index bd8f79e..24c6d6f 100644
--- a/node_modules/@searchkit/sdk/lib/esm/transporters/FetchClientTransporter.js
+++ b/node_modules/@searchkit/sdk/lib/esm/transporters/FetchClientTransporter.js
@@ -27,7 +27,8 @@ class FetchClientTransporter {
                 host = utils_1.getHostFromCloud(this.config.cloud);
             }
             const { index = this.config.index } = overrides;
-            const response = yield fetch(host + '/' + index + '/_search', {
+            const url = index ? host + '/' + index + '/_search' : host + '/_search'
+            const response = yield fetch(url, {
                 method: 'POST',
                 body: JSON.stringify(requestBody),
                 headers: Object.assign(Object.assign({ 'Content-Type': 'application/json' }, (((_a = this.config.connectionOptions) === null || _a === void 0 ? void 0 : _a.headers) || {})), (((_b = this.config.connectionOptions) === null || _b === void 0 ? void 0 : _b.apiKey)
